### DISCUSSION TITLE
Help for the dynamic island

### DISCUSSION THREAD

**Discussion by Shield31** [crissNb/Dynamic-Island-Sketchybar#26] [Q&A] [answered]
Hello, I come to ask you for help to make the dynamic island work properly. First of all, I would like to congratulate you for your work and your idea, I find it excellent.
I'm on macOs Monterey 12.6.3 on macbook pro mid 2012.
I manage to do the installation properly but I have a problem because the island appears to work for a few seconds and then no longer works. second and no longer works but I have found a way to make it work continuously by removing the reset.sh file and it works niquel but just one and I would like to understand why?

**Comment by crissNb** [comment]
Which "island" in particular do you have your issue with? Or does it not work in general?
Could you perhaps start sketchybar service from terminal via `sketchybar` and send me the log once it stops working?

**Reply by Shield31** [reply]
Hello, it does not work in general.
Here are pictures to show you.
![Sans titre](https://user-images.githubusercontent.com/127135629/223517264-69f01c60-a716-42fb-8aa9-afe720cae49a.png)
![Sans titre 1](https://user-images.githubusercontent.com/127135629/223517281-ec986f39-674f-422f-8dec-541703d038d9.png)

**Reply by crissNb** [reply, marked as answer]
That seems like locale problem to me. Could you perhaps try setting the locale to en_US manually? You can do this by adding this: `LC_NUMERIC="en_US.UTF-8"` line of code to the beginning of `volume_island.sh` and `brightness_island.sh` (or export the variable globally by adding it to one of the initializer scripts) temporarily to see if it fixes the problem.
 I'm currently away right now, so I won't be able to do it myself.

Edit: To add on, the other islands are being stopped most likely because brightness_island and/or volume_island does not finish normally due to the error. So essentially the other islands are just being blocked from displaying.
